phpunit.xml.dist

```diff
@@ -4,6 +4,7 @@
          bootstrap="./vendor/autoload.php"
          colors="true"
          verbose="true"
+         executionOrder="random"
```

Comment:
What do you think about keeping attributes sorted by name?

Reply:
thanks, done